src/sources/darwin/openWifiConnections.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env kmd
+exec defaults read /Library/Preferences/SystemConfiguration/com.apple.airport.preferences
```

Comment:
I believe the file should be `com.apple.airport.preferences.plist`

Reply:
Yes. 👍

I'll update this!